source/_integrations/roborock.markdown

```diff
@@ -127,6 +127,47 @@ Reset main brush consumable - The main brush/ roller is expected to be replaced
 
 Reset air filter - The air filter is expected to be replaced every 150 hours.
 
+### Actions
+
+#### Action `roborock.set_vacuum_goto_position`
+
+Go the specified coordinates.
+
+- **Data attribute**: `entity_id`
+  - **Description**: Only act on a specific robot.
+  - **Optional**: No.
+- **Data attribute**: `x_coord`
+  - **Description**: X-coordinate, integer value. The dock is located at x-coordinate 25500.
+  - **Optional**: No.
+- **Data attribute**: `y_coord`
+  - **Description**: Y-coordinate, integer value. The dock is located at y-coordinate 25500.
+  - **Optional**: No.
+
+#### Action `roborock.get_vacuum_current_position`
+
+Get the current position of the vacuum.
```

Comment:
good point, I would adjust it straight away.

Can I put vacuum_clean_rooms in this pr or should I make a new pr for it?

Reply:
You should make a new PR. That way if for some reason that one doesn't get merged in before a release that includes the last PR there isn't any confusion. One docs PR for one regular PR is best practice

Reply:
ok, but then it's probably best to create a branch with this as base instead of next, otherwise there will probably be problems with the merge because of the new actions section?